<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutKuzya</class>
 <widget class="QDialog" name="AboutKuzya">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Kuzya</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/menu/images/kuzya.png</normaloff>:/menu/images/kuzya.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="label" native="true">
     <layout class="QVBoxLayout" name="layout">
      <item>
       <widget class="QLabel" name="imgLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/common/Kuzya_about.png</pixmap>
        </property>
        <property name="buddy">
         <cstring>scrollAreaAuthors</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollAreaAuthors">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>617</width>
           <height>352</height>
          </rect>
         </property>
         <widget class="QLabel" name="textInfLabel">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>16</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollAreaAuthors</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
